test/milton/MiltonEvents.t.sol

```diff
@@ -0,0 +1,526 @@
+                openingFeeTreasuryAmount: TestConstants.ZERO, // openingFeeTreasuryAmount
+                iporPublicationFee: TestConstants.TC_IPOR_PUBLICATION_AMOUNT_18DEC, // iporPublicationFee
+                liquidationDepositAmount: TestConstants.TC_LIQUIDATION_DEPOSIT_AMOUNT_18DEC // liquidationDepositAmount
+            }), // money
+            block.timestamp, // openTimestamp
+            block.timestamp + TestConstants.SWAP_DEFAULT_PERIOD_IN_SECONDS, // endTimestamp, 28 days, PERIOD_28_DAYS_IN_SECONDS
+            MiltonTypes.IporSwapIndicator({
+                iporIndexValue: TestConstants.PERCENTAGE_3_18DEC, // iporIndexValue
+                ibtPrice: 1 * TestConstants.D18, // ibtPrice
+                ibtQuantity: 99670098970308907327800, // ibtQuantity
+                fixedInterestRate: TestConstants.PERCENTAGE_2_18DEC // fixedInterestRate, 2%
+            }) // indicator
+        );
+        miltonDai.itfOpenSwapPayFixed(
+            block.timestamp, // openTimestamp
+            TestConstants.USD_10_000_18DEC, // totalAmount
+            TestConstants.PERCENTAGE_1_18DEC, // acceptableFixedInterestRate, 1%
+            TestConstants.LEVERAGE_18DEC // leverage
+        );
+    }
+
+    function testShouldEmitEventWhenOpenPayFixedSwap6Decimals() public {
+        // given
+        MockTestnetToken usdtMockedToken = getTokenUsdt();
+        ItfIporOracle iporOracle = getIporOracleAsset(_userOne, address(usdtMockedToken), 0);
+        IpToken ipTokenUsdt = getIpTokenUsdt(address(usdtMockedToken));
+        MockCase0Stanley stanleyUsdt = getMockCase0Stanley(address(usdtMockedToken));
+        MiltonStorage miltonStorageUsdt = getMiltonStorage();
+        ItfMiltonUsdt miltonUsdt = getItfMiltonUsdt(
+            address(usdtMockedToken),
+            address(iporOracle),
+            address(miltonStorageUsdt),
+            address(_miltonSpreadModel),
+            address(stanleyUsdt)
+        );
+        ItfJosephUsdt josephUsdt = getItfJosephUsdt(
+            address(usdtMockedToken),
+            address(ipTokenUsdt),
+            address(miltonUsdt),
+            address(miltonStorageUsdt),
+            address(stanleyUsdt)
+        );
+        prepareApproveForUsersUsd(_users, usdtMockedToken, address(josephUsdt), address(miltonUsdt));
+        prepareMilton(miltonUsdt, address(josephUsdt), address(stanleyUsdt));
+        prepareJoseph(josephUsdt);
+        prepareIpToken(ipTokenUsdt, address(josephUsdt));
+        // when
+        _miltonSpreadModel.setCalculateQuotePayFixed(TestConstants.PERCENTAGE_4_18DEC); // 4%
+        vm.prank(_userOne);
+        iporOracle.itfUpdateIndex(address(usdtMockedToken), TestConstants.PERCENTAGE_3_18DEC, block.timestamp); // 3%, PERCENTAGE_3_18DEC
+        vm.prank(_liquidityProvider);
+        josephUsdt.itfProvideLiquidity(TestConstants.USD_28_000_6DEC, block.timestamp);
+        vm.prank(_userTwo);
+        vm.expectEmit(true, true, false, false);
```

Comment:
Why do you use vm.expectEmit(true, true, false, false) and not vm.expectEmit(true, true, true, true)